packages/integration-karma/test/api/sanitizeAttribute/x/xlink/xlink.html

```diff
@@ -1,5 +1,5 @@
 <template>
     <svg>
-        <use xlink:href="/foo"></use>
+        <use xlink:href={href}></use>
```

Comment:
Why modify this test?

Reply:
I was curious as well

Reply:
this is because of what [the test ](https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/api/sanitizeAttribute/index.spec.js)does. each test reset LWC.sanitizeAttribute to a spy, but because the static content is evaluated just once per component class, it never executes sanitizeAttribute again.